kuma/payments/jinja2/payments/management.html

```diff
@@ -0,0 +1,128 @@
+            </div>
+            <div class="manage-payments-help column-4">
+                <h4>{{_('Help')}}</h4>
+                <ul>
+                    <li>
+                        <a href="{{ url('recurring_payment_initial') }}" class="cta-link full-width">{{_('Monthly subscriptions FAQ')}} {% include 'includes/icons/arrows/arrow.svg' %}</a>
+                    </li>
+                    <li>
+                        <a href="{{ support_mail_link }}" class="cta-link cta-link email full-width">{{ support_mail }}{% include 'includes/icons/general/envelope.svg' %}</a>
+                    </li>
+                </ul>
+            </div>
+            <section class="column-half">
+                <h2>{{_('Subscriptions')}}</h2>
+                <p>
+                    {% trans url=support_mail_link, email=support_mail %}
+                        This is a list of your active subscriptions to MDN.
+                        If you wish to amend any of the details below please
+                        contact <a href="{{ url }}">{{ email }}</a>.
+                    {% endtrans %}
+                </p>
+            </section>
+        </article>
+        {% if request.user.stripe_customer_id and active_subscriptions %}
+            <section>
+                <h3>{{_('Monthly Subscription')}}</h3>
+                <table class="bordered-info">
+                    <thead>
+                        <tr>
+                            <th>{% include 'includes/icons/general/money-bill-alt.svg' %} {{_('Amount')}}</th>
+                            <th>{% include 'includes/icons/general/credit-card.svg' %} {{_('Card Number')}}</th>
+                            <th>{% include 'includes/icons/general/credit-card.svg' %} {{_('Expiry')}}</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td>${{ stripe_plan_amount }}</td>
+                            <td>**** **** **** {{ stripe_card_last4 }}</td>
+                            <td>02/19</td>
+                        </tr>
+                    </tbody>
+                </table>
+                {% if not cancel_request %}
+                    <form id="id_stripe_management_form" method="post">
+                        {% csrf_token %}
+                        <button class="button negative" type="button" toggle-confirmation>{{_('Cancel your subscription')}}</button>
+                        <div id="delete-confirmation" class="hidden" aria-hidden="true">
+                            <div class="confirmation column-8 alert danger">
+                                <h5>{{_('Are you sure?')}}</h5>
+                                <p>
+                                    {{_('You will have to set up a new subscription if you wish to resume making payments to MDN.')}}
```

Comment:
Just noticed this but, is the official brand name for MDN not MDN Web Docs? We should probably update all instances of MDN to MDN Web Docs to be consistent

Reply:
Good spot! I've updated a few places here, but it may be worth raising. I'll make a card for this.